src/lib/y2users/collision_checker.rb

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2issues"
+
+module Y2Users
+  # Internal class to check if there are collisions in config.
+  # This is not part of the stable API.
+  class CollisionChecker
+    include Yast::I18n
+
+    # Issue location describing the Group#gid attribute
+    GROUP_ID_LOC = "group:gid".freeze
```

Comment:
I know this `CollisionChecker` is designed to be generic. But at the end of the day, it's detecting a problem in the AutoYaST profile, so I would expect the location to point to the problematic part of the profile. That is, stuff like `autoyast:users` or `autoyast:groups`.

Reply:
According to the screenshot provided by @dgdavid, I'm not the only one who cannot make sense of the current locations, the Y2Issues presenter seems to be struggling with them as well.

Reply:
well, I think that it can also report issues on running system, but if it helps autoyast to group it as autoyast:users it is ok for me.